### PR TITLE
Write Workflows section of user manual

### PR DESCRIPTION
**Pull request by callunity**
GLT-2537

### CODE REVIEW

File: docs/user_manual/workflows.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Workflows are series of pre-defined steps that transition one or more items from a defined starting status to
+a defined end status. They represent a series of steps which would be completed in the lab in order to do some
+amount of work.
+
+{% assign sub = 1 %}
+{% assign figure = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Workflows Widget" %}
+
+To get to the Workflows widget, click on either the MISO logo or the Home tab in the top left of the page.
+This will take you to the Home page, which contains the Workflows widget. The Workflows (figure {{figure}})
```
Comment (djcooke) [inline docs/user_manual/workflows.md]: Could you add a subsection right at the top of the General Navigation section to describe the Home Page and how to get there?

Then remove the "how to get there" part from here and just say the widget is on the home screen

File: docs/user_manual/workflows.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Resuming an Incomplete Workflow" %}
+
+If you navigate away from the workflow page before completing the workflow, it is possible to re-enter the
+workflow where you left off.
+
+To resume an incomplete workflow, go to the Home page. In the workflows widget, click on the incomplete
```
Comment (djcooke) [inline docs/user_manual/workflows.md]: Capitalize "Workflows widget"

Comment (callunity) [reply docs/user_manual/workflows.md]: Updated

File: docs/user_manual/general_navigation.md [hunk 3]
```diff
@@ -5,8 +5,15 @@ section: 2
 ---
 
 
-
 {% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Home Screen" %}
```
Comment (djcooke) [inline docs/user_manual/general_navigation.md]: Needs added to the table of contents as well

Comment (callunity) [reply docs/user_manual/general_navigation.md]: Done

**Comment by callunity** [comment]
Interesting; it showed up for me with a hard refresh. I'll take a look at that link.

**Pull request merged**